py_zerox/zerox/models/modellitellm.py

```diff
@@ -65,55 +96,18 @@ async def completion(
         )
 
         try:
-            response = await self._make_request(messages, model)
-            return response
-        except Exception as err:
-            raise Exception(Messages.OPENAI_COMPLETION_ERROR.format(err))
+            response = await litellm.acompletion(model=self.model, messages=messages, **kwargs)
 
-    async def _make_request(
-        self,
-        messages: List[Dict[str, Any]],
-        model: str,
-        temperature: float = 0,
-    ) -> CompletionResponse:
-        """Makes a request to the OpenAI API for chat completions.
-
-        :param messages: A list of messages in the conversation.
-        :type messages: List[Dict[str, Any]]
-        :param model: The model to use for generating completions.
-        :type model: str
-        :param temperature: Controls the randomness of the output, defaults to 0
-        :type temperature: float, optional
-        :raises Exception: If the response status code is not 200.
-        :return: The response from the OpenAI API containing the completion content, input tokens, and output tokens.
-        """
-        async with aiohttp.ClientSession() as session:
-            async with session.post(
-                "https://api.openai.com/v1/chat/completions",
-                json={
-                    "messages": messages,
-                    "model": model,
-                    "temperature": temperature,
-                },
-                headers={
-                    "Authorization": f"Bearer {self.api_key}",
-                    "Content-Type": "application/json",
-                },
-            ) as response:
-                data = await response.json()
-
-                if response.status != 200:
-                    raise Exception(
-                        Messages.OPENAI_NON_200_RESPONSE.format(
-                            status_code=response.status, data=data
-                        )
-                    )
-
-                return CompletionResponse(
-                    content=data["choices"][0]["message"]["content"],
-                    input_tokens=data["usage"]["prompt_tokens"],
-                    output_tokens=data["usage"]["completion_tokens"],
+            ## completion response
+            response = CompletionResponse(
+                    content=response["choices"][0]["message"]["content"],
+                    input_tokens=response["usage"]["prompt_tokens"],
+                    output_tokens=response["usage"]["completion_tokens"],
                 )
+            return response
+        
+        except Exception as err:
+            raise Exception(Messages.COMPLETION_ERROR.format(err))
 
     async def _prepare_messages(
         self,
```

Comment:
OpenAI API -> LiteLLM API

Reply:
@xdotli, didn't got this one, can you elaborate.

Reply:
oh it's like a typo in comment. line 119

Reply:
"""Prepares the messages to send to the OpenAI API.
->
        """Prepares the messages to send to the LiteLLM API.

Reply:
@xdotli, my bad got it